enduro.toml

```diff
@@ -64,6 +64,10 @@ secret = "minio123"
 region = "us-west-1"
 bucket = "aips"
 
+# Change the taskqueue setting to your prefered preservation system, by default it is a3m.
+# [pres]
```

Comment:
I'd use "preservation" here because I don't think "pres" is a well known abbreviation.

Reply:
I'd think this section should be un-commented.